cli/src/api_client/mod.rs

```diff
@@ -172,7 +177,14 @@ impl ApiClient {
             },
         }
         .call_api()
-        .await
+        .await;
+
+        if let Err(ref e) = result {
+            log::error!("Error during API call: {}", e);
+            return Err(anyhow::anyhow!(e.to_string()));
+        }
+
+        result
```

Comment:
nit: remove some redundancy

```suggestion
        if result.is_err() {
            log::error!("Error retrieving quarantine configuration");
        }

        result
```